docs/api.md

```diff
@@ -966,93 +1075,70 @@ Example Response Body:
-| limit      | The maximum number of standard actions to retrieve.             |
-| product    | The product of the standard action, e.g. `perpetual` or `solo`. |
+Gets the perpetual and solo standard actions for a particular user.
 
-Example Response Body:
-
-```json
-{
-  "standardActions": [
-    {
-      "uuid": "b95fa3fc-84a7-46f1-9ce0-1eca1b144117",
-      "type": "DEPOSIT",
-      "owner": "0x014be43bf2d72a7a151a761a1bd5224f7ad4973c",
-      "transferAmount": "33720746949513441",
-      "price": "6227979999899999000000",
-      "market": "PBTC-USDC",
-      "side": "LONG",
-      "orderNumber": "956855500050000",
-      "confirmedAt": "2020-02-27T23:10:31.000Z",
-      "createdAt": "2020-02-27T23:11:31.758Z",
-      "updatedAt": "2020-02-27T23:11:31.778Z",
-      "product": "perpetual",
-    },
-  ]
-}
-```
 
-### Standard Action Response Object
+#### Standard Action Response Object
 
 | Field Name     | Description                                                   |
 |----------------|---------------------------------------------------------------|
 | uuid           | The unique id for the action.                                 |
 | owner          | The wallet address of the user.                               |
-| type           | The type of standard action e.g. `DEPOSIT`.                   |
+| type           | The type of standard action e.g. `OPEN` or `ACCOUNT_SETTLE`.  |
+| asset          | The asset eg `USDC` for deposit or withdraw                   |
 | market         | The perpetual market, e.g. `PBTC-USDC`.                       |
 | side           | The side for the standard action e.g. `LONG`, `SHORT`.        |
-| transferAmount | The amount in settlement token that is transferred.           |
+| transferAmount | The amount transferred in settlement token                    |
+| tradeAmount    | The amount traded. i.e. the base token amount in a trade      |
 | price          | The price in settlement token.                                |
 | orderNumber    | Number used for ordering the standard actions.                |
+| product        | The product type, e.g. `perpetual`                            |
 | updatedAt      | The ISO 8601 date and time the standard action was updated.   |
 | createdAt      | The ISO 8601 date and time the standard action was created.   |
 | confirmedAt    | The ISO 8601 date and time the standard action was confirmed. |
-| product        | The product type, e.g. `perpetual` or `solo`.                 |
+| transactionHash| The transaction corresponding to this standard action         |
+| feeAmount      | The fee amount charged                                        |
+| feeAsset       | The asset of the `feeAmount` eg `USDC`                        |
+| payoutAmount   | The amount refunded to the user when maker fee is negative    |
+| pnl            | Currently not set in the standard action for perpetual        |                                               
```

Comment:
nit `The PnL for the corresponding position. Currently not set in the standard action for perpetual.`

Reply:
fixed